promql/engine.go

```diff
@@ -2743,7 +2751,10 @@ func (ev *evaluator) aggregation(e *parser.AggregateExpr, grouping []string, par
 			if s.H != nil {
 				group.hasHistogram = true
 				if group.histogramValue != nil {
-					group.histogramValue.Add(s.H)
+					_, err := group.histogramValue.Add(s.H)
+					if err != nil {
+						panic(err)
+					}
 				}
```

Comment:
We should not panic in the engine, instead ignore the value in the addition. Possibly throw a warning.